spec/support/data_helpers.rb

```diff
@@ -6,6 +6,29 @@ def create_active_school(*args)
       school_creator.process_new_configuration!
     end
   end
+
+  def create_all_calendar_events
```

Comment:
This method was a service in the main application, but it's never going to be used again by the main app, so moved here in to a test support file